ibis/backends/base/sql/alchemy/query_builder.py

```diff
@@ -321,21 +318,19 @@ def reduce_union(left, right, distincts=iter(self.distincts)):
         return functools.reduce(reduce_union, selects)
 
 
-class AlchemyQueryBuilder(QueryBuilder):
-
+class AlchemyCompiler(Compiler):
+    translator = AlchemyExprTranslator
```

Comment:
Could we use consistent naming here? Some of the properties have the `_class` prefix some not. Another alternative could be to use CamelCase, like `AlchemyCompiler.Select()` or `AlchemyCompiler.TableSetFormatter()`

Of course this can be deferred to a follow-up.

Reply:
Yes, good point, thanks for the review. I fixed the `_compiler` here, but I prefer to take care of this in a follow up. The inconsistency already exists in master, and this PR is already huge to add more stuff. I created #2824 for this.